overlord/servicestate/quota_handlers.go

```diff
@@ -222,15 +222,17 @@ func ensureSnapServicesForGroup(st *state.State, grp *quota.Group, opts *ensureS
 
 		// drop disabled services from the startup ordering
 		startupOrderedMinusDisabled := make([]*snap.AppInfo, 0, len(startupOrdered)-len(disabledSvcs))
+		explicitServices := make([]string, 0, len(startupOrderedMinusDisabled))
```

Comment:
hmm, after learning what explicitServices is used for, do we want inactive services to be started when they are put into a quota group? I think not, I think actually the default behavior should be used such that `explicitServices` is nil for this, so that if I have an inactive service in my snap and I add that snap to a quota group, the service is not started.

Reply:
I think that what you are saying makes sense. I only added this here because I got the tests failing otherwise. Let's sync later.

Reply:
@pedronis but there's a distinction between disabled and inactive, if I have a service which is _enabled_ but inactive, it's unexpected to me that that service would be started up after I put my snap into a quota group. 

Perhaps we should chat in/after the SU

Reply:
yes, I misremembered the new code when I made my comment. Anyway there's probably the reverse problem too, related to disabled. So I think what we want is a combination of suggestions. Let's chat after the standup.

Reply:
Changed the code as agreed.